__tests__/app.test.js

```diff
@@ -0,0 +1,42 @@
+const request = require('supertest')
+const {app} = require('../app')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data')
+
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+
+describe('GET /api/unrecognisable-endpoints', ()=>{
+    test('404: responds with not found when given an unrecognisable endpoint', () =>{
```

Comment:
we only need to test this if we are sending a message back! - this is an express response otherwise so no need to test